SWLOR.Game.Server/Core/Bioware/BiowarePosition.cs

```diff
@@ -91,6 +91,9 @@ public static float GetRelativeFacing(uint o1, uint o2)
             // X/Y so that we're taking angle relative to the Y axis (X is opposite, Y adjacent)
             var angle = (float)Math.Atan(diffX / diffY);
 
+            // @@@ PROBABLE BUG - this should probably be NWScript.atan not Math.Atan as the latter
```

Comment:
Let's go ahead and switch the Math.Atan call back to the NWScript.atan call if this is causing an issue.

Reply:
It's not causing an issue - but currently nothing calls this method!  Since I have no code set up to test it I didn't want to mess with it today, but I can change it if you like.